libs/sdk-core/src/models.rs

```diff
@@ -750,6 +751,16 @@ pub struct ReceivePaymentRequest {
     pub cltv: Option<u32>,
 }
 
+impl ReceivePaymentRequest {
+    pub fn from_amount_and_description(amount_msat: u64, description: String) -> Self {
```

Comment:
TBH I do understand the reason for this but I think it is not needed as the explicit way of passing the parameters is already short and easy enough. Unnecessary maintenance overhead IMO.